pkg/ovs/openflow/utils.go

```diff
@@ -1135,7 +1133,11 @@ func getFlowModAction(flowMod *openflow15.FlowMod) string {
 }
 
 func FlowModToString(flowMod *openflow15.FlowMod) string {
-	return fmt.Sprintf("%s, %s %s", getFlowModBaseString(flowMod), getFlowModMatch(flowMod), getFlowModAction(flowMod))
+	return strings.Join([]string{fmt.Sprintf("%s,", getFlowModBaseString(flowMod)), getFlowModMatch(flowMod), getFlowModAction(flowMod)}, " ")
```

Comment:
don't feel such change is necessary, no real benefit by the change but adds extra diff